.evergreen/run-mockupdb-tests.sh

```diff
@@ -2,19 +2,17 @@
 
 set -o xtrace
 set -o errexit
-
+cd $PROJECT_DIRECTORY
 . .evergreen/utils.sh
 
 ${PYTHON_BINARY} setup.py clean
-cd ..
 
 createvirtualenv ${PYTHON_BINARY} mockuptests
 trap "deactivatei, rm -rf mockuptests" EXIT HUP
 
 # Install PyMongo from git clone so mockup-tests don't
 # download it from pypi.
 python -m pip install ${PROJECT_DIRECTORY}
-
-git clone https://github.com/ajdavis/pymongo-mockup-tests.git
-cd pymongo-mockup-tests
-python setup.py test
+python -m pip install mockupdb
```

Comment:
The old tests install mockupdb@master. Let's do the same here:
```
python -m pip install --upgrade 'https://github.com/ajdavis/mongo-mockup-db/archive/master.zip'

Reply:
Done.